src/client/app/frameworks/core/services/app.service.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@angular/core';
+
+// app
+import { AnalyticsService } from '../../../frameworks/analytics/index';
+import { LogService } from './log.service';
+import { Config } from '../utils/config';
+import { MultilingualService } from '../../../frameworks/i18n/index';
+
+@Injectable()
+export class AppService {
```

Comment:
Just to make it clear, since I haven't explained it.

`AppService` and `NSAppService` were created because `nativescript-dev-webpack` didn't like the `NSAppComponent` that inherited from `AppComponent` but wasn't real a `@Component`.
It's only supported to get injected in the `AppComponent`, the idea is to setup `NativeScript` listeners and more.